ledger/src/blocktree_db.rs

```diff
@@ -374,6 +386,33 @@ impl TypedColumn for columns::DeadSlots {
     type Type = bool;
 }
 
+impl Column for columns::DuplicateSlots {
```

Comment:
Can we use some templating to avoid this duplication with other slot-based columns?

Reply:
Updatedd